src/FixedDataTable.js

```diff
@@ -577,9 +577,9 @@ class FixedDataTable extends React.Component {
       this._didScrollStart();
```

Comment:
Unrelated to this PR, but wouldn't this call need the updated props?

Reply:
Could you clarify the question?  I'm not sure I follow

Reply:
Since the call is within `componentWillRecieveProps(nextProps)`, I was wondering if we require `nextPros` to be passed to `_didScrollStart` ?

Reply:
Ah yes, that makes sense.